app/views/conferences/_tickets.html.haml

```diff
@@ -14,12 +14,12 @@
             %p.text-center
               %i.fa.fa-ticket.fa-5x
             .caption
-              %h3.text-center
+              %h3.text-center{ style: 'word-wrap: break-word' }
                 = ticket.title
               -if ticket.description.present?
-                = markdown(ticket.description)
+                %div{ style: 'word-wrap: break-word' }
```

Comment:
Same here.

Reply:
Sure! I'm on it

Reply:
@Astradeni feel free to reach out, if you need help with where to put this.

Reply:
I probably will... I'm thinking maybe assets/stylesheets?